src/Symfony/Component/VarDumper/Dumper/ServerDumper.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarDumper\Dumper;
+
+use Symfony\Component\VarDumper\Cloner\Data;
+use Symfony\Component\VarDumper\Server\DumpServer;
+
+/**
+ * ServerDumper forwards serialized Data clones to a server.
+ *
+ * @author Maxime Steinhausser <maxime.steinhausser@gmail.com>
+ */
+class ServerDumper implements DataDumperInterface
+{
+    private $host;
+    private $wrappedDumper;
+    private $swallow;
+    private $contextProviders;
+    private $socket;
+
+    /**
+     * @param string|null              $host             The server host or null to read it from the VAR_DUMPER_SERVER
+     *                                                   env var
+     * @param DataDumperInterface|null $wrappedDumper    A wrapped instance used whenever we failed contacting the
+     *                                                   server, or if swallow is false
+     * @param bool                     $swallow          Whether to use or not the wrapped dumper instance on normal
+     *                                                   use
```

Comment:
not sure we need a dedicated line for one word ;) (same above btw)

Reply:
fixed